bio/gatk3/indelrealigner/wrapper.py

```diff
@@ -35,6 +56,6 @@
     " {input_known_string}"
     " {bed}"
     " --targetIntervals {input_target_intervals}"
-    " -o {snakemake.output}"
+    " -o {output_bam}"
```

Comment:
To avoid the explicit get operation from above, you can simply make this:
```suggestion
    " -o {snakemake.output.bam}"
```

Reply:
While we are at it, I would do the same for `snakemake.input.bam`, `snakemake.input.ref` and `snakemake.input.target_intervals`. Nothing is ever done to parse them any further, so we can simply paste them in here.